schemas/error.yml

```diff
@@ -29,6 +29,12 @@
       description: >
         Error code describing the error.
 
+    - name: type
+      level: core
+      type: keyword
+      description: >
```

Comment:
Could you add an example under `example` key? Would also be great to have at least 2 examples in the description. The reason I'm insisting on this is that `type` fields are always these "well we could fit everything in here" and its always hard to explain what is in. Having example should help.

Reply:
Added an `example` field but could not come up with a good second example for the description